pkg/validation/microdeposits/micro_deposits.go

```diff
@@ -75,6 +82,11 @@ func createMicroDeposits(
 	return micro, nil
 }
 
+func skipDebit(cfg config.MicroDepositWithdraw, acct customers.Account) bool {
```

Comment:
I suggest removing `skipDebit` and being explicit:
```go
if !cfg.Withdraw.FromSavingsAccount && dest.Account.Type != customers.Savings {
 return micro, nil    
}
```

To figure out what `skipDebit` does, I had to look into its implementation.